scripts/misc/species_list.pl

```diff
@@ -146,6 +146,7 @@ =head1 CONTACT
     $species_list{$s_name}{'count'} = round_count($count_var);
 
     my $label_name = ucfirst($s_name);
+       $label_name =~ s/gca[0-9v]+//g;	# remove any gca from name 
```

Comment:
NB: This would impact any species that contains "gcav" in its scientific name. We don't have any such species in Ensembl (and from a quick Google search, there doesn't seem to be any such species.

Still, I would suggest the following to ensure we only strip out the proper GCA from the species name.

```suggestion
       $label_name =~ s/_gca[0-9]{9}[v0-9]*+$//g;	# remove any gca from name 
```